src/momento/errors/error_converter.py

```diff
@@ -0,0 +1,87 @@
+from typing import Dict, List, Optional, Tuple, Type, Union
+
+import grpc
+from grpc.aio import Metadata
+
+from momento.errors import (
+    AlreadyExistsException,
+    AuthenticationException,
+    BadRequestException,
+    CancelledException,
+    FailedPreconditionException,
+    InternalServerException,
+    InvalidArgumentException,
+    LimitExceededException,
+    MomentoErrorTransportDetails,
+    MomentoGrpcErrorDetails,
+    NotFoundException,
+    PermissionDeniedException,
+    SdkException,
+    ServerUnavailableException,
+    TimeoutException,
+    UnknownException,
+    UnknownServiceException,
+)
+
+grpc_status_to_exception: Dict[grpc.StatusCode, Type[SdkException]] = {
+    grpc.StatusCode.INVALID_ARGUMENT: InvalidArgumentException,
```

Comment:
Looks like `grpc.StatusCode.Internal` is missing from this mapping. It maps to `InternalServerException`.

Reply:
Thanks. This is missing from the [.NET SDK as well](https://github.com/momentohq/client-sdk-dotnet/blob/main/src/Momento.Sdk/Exceptions/CacheExceptionMapper.cs) (which I based this off of)

Reply:
Fixed in [ed5e6cf](https://github.com/momentohq/client-sdk-python/pull/179/commits/ed5e6cf5b0afdb30a14c1960bfea18cc9fe71152)